.github/workflows/format-code.yml

```diff
@@ -48,12 +48,9 @@ jobs:
           git checkout -b "$HEAD_REF" "origin/$HEAD_REF"
 
       - name: 'Format code'
-        run: |
-          # Use prettier to format the added or modified Markdown files in the PR (if any)
-          jq -r '.issue.pull_request.url | . += "/files"' "$GITHUB_EVENT_PATH" | \
-            xargs curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' --url | \
-            jq -c '.[] | select(.status == "added" or .status == "modified") | select(.filename | match("\\.(md|json)$")) | .filename' | \
-            xargs npx prettier@2.0.4 --write
+        run: ./bin/format.sh
+        env:
+          EXERCISM_PRETTIER_VERSION: '2.1.2'
```

Comment:
I wonder if we should set this as a secret so that it can't possibly get out of sync ever.